en/docs/administer/configure-approvals-for-choreo-workflows.md

```diff
@@ -2,14 +2,30 @@
 
 Choreo allows you to configure approval processes for specific workflows within the platform. An approval process for a workflow ensures that critical or sensitive changes are properly managed and controlled.
 
-Choreo currently allows you to configure approvals for environment promotion workflows, with support for API subscription approvals coming soon.
+Choreo currently allows you to configure approvals for environment promotion and API subscription workflows.
+
+Configuring approvals for environment promotion allows authorized users to control components being promoted to a critical/production environment. 
+
+Configuring approvals for the API subscription workflow allows you to create subscription plans that require approval before being activated. This feature allows you to control access to APIs by requiring administrative review and authorization of subscriptions before they become active.
 
 ## Permissions to review and respond to approval requests
 
-To review and respond to approval requests, a user must have the following permissions. Administrators must ensure that users designated to review and respond to approval requests have these permissions:
+Click the respective tab for details on permissions depending on the workflow for which you want to configure approvals:
+
+=== "Environment promotion"
+
+     To review and respond to environment promotion approval requests, a user must have the following permissions. Administrators must ensure that users designated to review and respond to approval requests have these permissions:
+
+      - **WORKFLOW-MANAGEMENT**: Grants access to view and approve workflow requests. Each workflow type has a separate permission.
+      - **PROJECT-MANAGEMENT**: Grants access to view and approve workflow requests. This is the same permission used to update or delete projects.
+
+=== "API subscription"
 
-  - **WORKFLOW-MANAGEMENT**: Grants access to view and approve workflow requests. Each workflow type has a separate permission.
-  - **PROJECT-MANAGEMENT**: Grants access to view and approve workflow requests. This is the same permission used to update or delete projects.
+     To review and respond to API subscription approval requests, a user must have the following permissions. Administrators must ensure that users designated to review and respond to approval requests have these permissions:
+
+      - **WORKFLOW-MANAGEMENT**: Grants access to view and approve workflow requests. Each workflow type has a separate permission.
```

Comment:
`**Approve API subscriptions**: Grants access to approve or reject API subscription requests.`
This needs to be under WORKFLOW-MANAGEMENT set, doesn't it? 

And re: "Environment promotion" I think we need to have a similar way to indicate "Approve component promotion requests" too